Source/carthage/BuildVersion.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  BuildVersion.swift
+//  Carthage
+//
+//  Created by Rodrigo Garcia on 10/15/16.
+//  Copyright © 2016 Carthage. All rights reserved.
+//
+
+import Foundation
+import CarthageKit
+import ReactiveCocoa
+import ReactiveTask
+import Result
+import Tentacle
+
+public struct CarthageVersion{
+	
+	public let localVersion:SemanticVersion
+	public let remoteVersion:SemanticVersion
```

Comment:
Done!

BTW, I activated SwiftLint and got ˜15000 issues :D in the project. I think that it isn't being used.